packages/vault/src/index.ts

```diff
@@ -82,28 +130,21 @@ export class VaultEnvVars {
     } else {
       // developer's local machine
       try {
-        const vaultTokenFile = await fs.readFile(path.join(os.homedir(), '.vault-token'), {
-          encoding: 'utf8'
-        })
-        if (vaultTokenFile) {
-          return vaultTokenFile
-        } else if (VAULT_AUTH_GITHUB_TOKEN) {
-          console.log(`You are not logged in, logging you in...`)
-          const token = await fetch<Token>(`${VAULT_ADDR}/auth/github/login`, {
-            method: 'POST',
-            headers: { 'Content-type': 'application/json' },
-            body: JSON.stringify({ token: VAULT_AUTH_GITHUB_TOKEN })
-          })
-          return token.auth.client_token
+        const token = await this.handleLocalToken()
+        return token
+      } catch {
+        if (VAULT_AUTH_GITHUB_TOKEN) {
+          try {
```

Comment:
**suggestion:** I don't think we need to wrap line 138 to 139 in a try catch as the catch block is throwing the same error that line 138 would throw